### ISSUE TITLE
Add ability to blacklist/revoke JWT tokens

### ISSUE THREAD

**Issue by benpankow** [HackIllinois/api#311]
Right now, JWTs cease to work only after they expire. It would be nice to have the ability to specifically revoke a JWT or JWTs for a particular user.

**Comment by benpankow** [comment]
Relevant code: https://github.com/HackIllinois/api/blob/e20175423cdf3ad50d69231ab72dfd0cca0c2c6b/gateway/utils/jwt.go#L24
